api/HTMLInputElement.json

```diff
@@ -1588,10 +1588,10 @@
               "version_added": "≤18"
             },
             "firefox": {
-              "version_added": "16"
+              "version_added": false
```

Comment:
Indeed not supported per http://mdn-bcd-collector.appspot.com/tests/api/HTMLInputElement/weight.

But what is `input.weight`, was it ever a thing?

I suspect not, since a `width` entry is missing I suspect this is just a typo of that which was never caught before. Can you rename the entry and update the data with collector results? It should probably be the same as for `height`, but not necessarily, it seems plausible `width` is actually older than `height`.

Reply:
I've created #7671 to resolve this issue.